### ISSUE TITLE
Gradient in the default visualizer

### ISSUE THREAD

**Issue by danimesq** [nt1m/HTML5AudioPlayer#69]

**Comment by nt1m** [comment]
What should it look like? The original visualizer had a green gradient that I removed.

**Comment by danimesq** [comment]
Yes, like the original I'd seen today. But with the actual color. And the gradient can be different, but gradient <3
